### PR TITLE
docs: Make Heroku button and setup instructions use 'latest' branch i…

### PR DESCRIPTION
**Pull request by michielbdejong**
…nstead of 'master'.

As discussed with @dappelt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ILP wallet with hosted ledger and connector instances
 [![circle][circle-image]][circle-url] [![Known Vulnerabilities][snyk-image]][snyk-url] [![Deploy to Heroku][heroku-deploy-image]][heroku-deploy-url]
 
 [heroku-deploy-image]: https://www.herokucdn.com/deploy/button.svg
-[heroku-deploy-url]: https://heroku.com/deploy?template=https://github.com/interledgerjs/ilp-kit/tree/master
+[heroku-deploy-url]: https://heroku.com/deploy?template=https://github.com/interledgerjs/ilp-kit/tree/lastest
```
Comment (dappelt) [inline README.md]: There is a typo here. **lastest** -> **latest**

File: docs/SETUP.md [hunk 2]
```diff
@@ -122,6 +122,7 @@ dependencies with the following commands:
 $ cd # start in your home folder
 $ git clone https://github.com/interledgerjs/ilp-kit
 $ cd ilp-kit
+$ git checkout lastest # the 'master' branch is not always stable, safer to use the 'latest' branch
```
Comment (dappelt) [inline docs/SETUP.md]: typo here as well

**Comment by michielbdejong** [comment]
It actually seems like heroku is still deploying the master branch, investigating.

File: circle.yml [hunk 3]
```diff
@@ -21,12 +21,12 @@ test:
 
 deployment:
   production:
-    branch: master
+    branch: latest
```
Comment (michielbdejong) [inline circle.yml]: This changes the frequency of adding a build to https://hub.docker.com/r/interledgerjs/ilp-kit/tags/

File: circle.yml [hunk 4]
```diff
@@ -21,12 +21,12 @@ test:
...
     commands:
       # Push Docker image tagged latest and tagged with commit descriptor
       - sed "s/<AUTH>/${DOCKER_TOKEN}/" < "dockercfg-template" > ~/.dockercfg
       - docker push interledgerjs/ilp-kit:latest
-      - docker push interledgerjs/ilp-kit:"$(git describe)"
+      - docker push interledgerjs/ilp-kit:"$(git describe --tags)"
```
Comment (michielbdejong) [inline circle.yml]: See [commit comment](https://github.com/interledgerjs/ilp-kit/commit/febbca2a8beabcddcdd327d657e45f98a2d388a1)

File: integration-test/package.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (dappelt) [inline integration-test/package.json]: This file should not be committed.

File: package.json [hunk 6]
```diff
@@ -38,7 +38,7 @@
     "test": "touch src/theme/variables-custom.scss ; karma start",
     "test-api": "better-npm-run test-api",
     "test-mocha": "NODE_ENV=test ./node_modules/.bin/mocha api/test",
-    "integration": "integration-loader && integration all",
+    "integration": "integration-loader && (integration setup > /dev/null 2>&1) && integration test",
```
Comment (dappelt) [inline package.json]: I find some of the output of `integration setup` quite useful (e.g. to see which versions were installed) and, hence, I would not redirect all output to /dev/null.

If you want to make the build output less chatty I propose to change the `build` command.

**Comment by dappelt** [comment]
Also, please squash the commits.

File: package.json (hunk 6 above)
Comment (michielbdejong) [inline package.json]: ok, I removed this commit from the PR

**Pull request merged**